setup.py

```diff
@@ -29,7 +29,7 @@
 # This allows us to inject a custom version into setuptools using the
 # environment variable: WTM_VERSION. If it is not specified, a default value is
 # used, but you will not be able to run sdist in this case.
-DEFAULT_VERSION = "dev"
+DEFAULT_VERSION = "0.dev"
```

Comment:
Why does this need to be changed? I have not run test, but I never changed the default. I wonder if it will introduce subtle regressions?

Reply:
Changing it from "dev" to "0.dev" did not have a noticeable effect in my setup. Thus, I'm fine with changing it.

I assume, it comes from some semantic versioning naming conventions, which require it to start with an integer. "0.devN" seems more common.

I wonder if it is also related to the README?

https://github.com/whotracksme/whotracks.me/blob/ebe5b85126788b94251b411a58e3ed2883d4b1fa/README.md?plain=1#L121

https://github.com/whotracksme/whotracks.me/blob/ebe5b85126788b94251b411a58e3ed2883d4b1fa/README.md?plain=1#L145

But the commands from the README still seem to work for me.